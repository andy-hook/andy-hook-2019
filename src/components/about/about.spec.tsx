import React from "react"

import About from "./about"
import { shallowWithTheme } from "@test-utils"

describe("<About />", () => {
  it("renders correctly", () => {
    const tree = shallowWithTheme("dark", <About />)
    expect(tree).toMatchSnapshot()
  })
})
