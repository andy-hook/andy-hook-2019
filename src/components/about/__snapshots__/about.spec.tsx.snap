// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<About /> renders correctly 1`] = `
<div
  className="sc-EHOje PRAcs"
>
  <div
    className="sc-bdVaJa bUDcPf"
  >
    <div
      className="sc-htpNat dzobEj"
    >
      <h1
        className="sc-bZQynM jBjNHF"
      >
        Overline text
      </h1>
      <h2
        className="sc-gzVnrw cpBZxR"
      >
        TitleÂ text
      </h2>
      <div
        className="sc-htoDjs khBefK"
      />
    </div>
  </div>
  <footer
    className="sc-ifAKCX hSmABr"
  >
    <div
      className="sc-bdVaJa bUDcPf"
    >
      <div
        className="sc-htpNat dzobEj"
      >
        I am a footer
      </div>
    </div>
  </footer>
</div>
`;
